src/Components/Auction/__stories__/AuctionRegistrationModal.story.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react"
+import { storiesOf } from "storybook/storiesOf"
+import { AuctionRegistrationModal } from "../AuctionRegistrationModal"
+
+const submitHandler = p => {
+  setTimeout(() => {
+    p.setSubmitting(false)
+    alert("Your Submission Callback Here")
+  }, 1000)
+}
+storiesOf("Components/Auction/AuctionRegistrationModal", module).add(
+  "Default",
+  () => {
+    return (
+      <>
+        <AuctionRegistrationModal
+          onSubmit={submitHandler}
```

Comment:
This is very cool! I like how you're able to override the `onSubmit` behavior within the story :+1:

Reply:
This was a side effect of initially writing this to use a relay mutation- as I was backing it out I decided to move away from the tie to force, and figured that a future application might use relay to pass its `commitMutation` function (etc) in.